Make error

When I run `make` on latest master `97fa0d343728af0301e5d371ecc119eb9e4f47f6`, I got the following error:
```
root@vagrant-ubuntu-trusty-64:/opt/packer-provisioner-host-command# make
rm -f /opt/go/bin/packer-provisioner-host-command
go get github.com/flynn/go-shlex
go get github.com/mitchellh/packer
go build -o /opt/go/bin/packer-provisioner-host-command -a
../go/src/github.com/mitchellh/packer/common/step_connect_ssh.go:4:2: cannot find package "code.google.com/p/go.crypto/ssh" in any of:
	/usr/lib/go/src/pkg/code.google.com/p/go.crypto/ssh (from $GOROOT)
	/opt/go/src/code.google.com/p/go.crypto/ssh (from $GOPATH)
../go/src/github.com/mitchellh/packer/common/step_create_floppy.go:5:2: cannot find package "github.com/mitchellh/go-fs" in any of:
	/usr/lib/go/src/pkg/github.com/mitchellh/go-fs (from $GOROOT)
	/opt/go/src/github.com/mitchellh/go-fs (from $GOPATH)
../go/src/github.com/mitchellh/packer/common/step_create_floppy.go:6:2: cannot find package "github.com/mitchellh/go-fs/fat" in any of:
	/usr/lib/go/src/pkg/github.com/mitchellh/go-fs/fat (from $GOROOT)
	/opt/go/src/github.com/mitchellh/go-fs/fat (from $GOPATH)
../go/src/github.com/mitchellh/packer/common/multistep_debug.go:5:2: cannot find package "github.com/mitchellh/multistep" in any of:
	/usr/lib/go/src/pkg/github.com/mitchellh/multistep (from $GOROOT)
	/opt/go/src/github.com/mitchellh/multistep (from $GOPATH)
make: *** [build] Error 1
```
I had to edit `Makefile` and add the following under `deps`:
```
go get github.com/mitchellh/multistep
go get github.com/mitchellh/go-fs
go get code.google.com/p/go.crypto/ssh
```

I'm totally new to Go, so please let me know if this is a real issue, or I was doing something wrong.

Thanks!

--- a/Makefile
+++ b/Makefile
@@ -6,6 +6,9 @@ clean:
 deps:
 	go get github.com/flynn/go-shlex
 	go get github.com/mitchellh/packer
+	go get github.com/mitchellh/multistep
+	go get github.com/mitchellh/go-fs
+	go get code.google.com/p/go.crypto/ssh
 
 build:
 	go build -o $(GOPATH)/bin/packer-provisioner-host-command -a

